infrastructure/metadata/infrastructure/server/webtransport-h3.https.sub.any.js.ini

```diff
@@ -1,28 +1,30 @@
 [webtransport-h3.https.sub.any.html]
-  [WebTransport server should be running and should handle a bidirectional stream]
-    expected:
-      if product == "chrome": PASS
-      FAIL
-
-[webtransport-h3.https.sub.any.window.html]
+  disabled:
+    if product == "firefox": times out
```

Comment:
I thought this how to be "TIMEOUT" yet seemingly this works 🤷🏻‍♀️

Reply:
ah yeah, I clearly didn't look closely enough! I think we historically _haven't_ done that (and just eaten the run time cost, because really the infrastructure job doesn't run a huge amount), partly so we notice when things progress again—but in this case where we are likely to progress it in the near future via https://github.com/web-platform-tests/wpt/pull/39322 it doesn't seem such an issue.